flujo/application/core/executor_core.py

```
@@ -743,6 +755,128 @@ async def _execute_complex_step(
+            )
+
+            # Execute
+            await self.execute(frame)
+
+        except (PausedException, PipelineAbortSignal) as control_flow_err:
+            # Control-flow exceptions from background tasks are logged but don't propagate
+            # since the main pipeline has moved on. This is intentional for background mode.
+            telemetry.logfire.warning(
+                f"Background task '{getattr(step, 'name', 'unknown')}' raised control-flow signal: {control_flow_err}"
+            )
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)